src/main/java/de/rytrox/bedwars/utils/ShopUtils.java

```diff
@@ -0,0 +1,345 @@
+        ItemStack none = new ItemStackBuilder(Material.WHITE_STAINED_GLASS_PANE)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&f"))
+                .toItemStack();
+        ItemStack sellected = new ItemStackBuilder(Material.RED_STAINED_GLASS_PANE)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&f"))
+                .toItemStack();
+        ItemStack rush = new ItemStackBuilder(Material.STICK)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&b&lRush"))
+                .toItemStack();
+        ItemStack blocks = new ItemStackBuilder(Material.SANDSTONE)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&b&lBlöcke"))
+                .toItemStack();
+        ItemStack armor = new ItemStackBuilder(Material.IRON_CHESTPLATE)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&b&lRüstung"))
+                .toItemStack();
+        ItemStack swords = new ItemStackBuilder(Material.IRON_SWORD)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&b&lSchwerter"))
+                .toItemStack();
+        ItemStack tools = new ItemStackBuilder(Material.IRON_PICKAXE)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&b&lWerkzeuge"))
+                .toItemStack();
+        ItemStack potions = new ItemStackBuilder(Material.POTION)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&b&lTränke"))
+                .toItemStack();
+        ItemStack gadgets = new ItemStackBuilder(Material.TNT)
+                .setDisplayName(ChatColor.translateAlternateColorCodes('&', "&b&lGadgets"))
+                .toItemStack();
+        for(int i = 0; i < inventory.getSize(); i++) inventory.setItem(i, none);
+        inventory.setItem(1, rush);
+        inventory.setItem(2, blocks);
+        inventory.setItem(3, armor);
+        inventory.setItem(4, swords);
+        inventory.setItem(5, tools);
+        inventory.setItem(6, potions);
+        inventory.setItem(7, gadgets);
+        inventory.setItem(9 + tab, sellected);
+        return inventory;
+    }
+
+    // summons the Shop Villager
+    public static void summonShopVillager(Location location) {
+        Villager villager = (Villager) Objects.requireNonNull(location.getWorld()).spawnEntity(location, EntityType.VILLAGER);
+        villager.setCustomName(ChatColor.translateAlternateColorCodes('&', "&9&lShop"));
+        villager.setCustomNameVisible(true);
+        villager.setAI(false);
+        villager.setSilent(true);
+    }
+
+    // kills all "Shop Villagers" in a custom radius
+    public static void killShopVillagers(Location location, int radius) {
+        Objects.requireNonNull(location.getWorld()).getNearbyEntities(location, radius, radius, radius)
+                .stream()
+                .filter((entity) -> entity instanceof Villager)
```

Comment:
```suggestion
                .filter(Villager.class::isInstance)
```

Das wäre eine ordentlichere Lösung